### ISSUE TITLE
Requires Screen Recording Permission for macOS Catalina (10.15) and later

### ISSUE THREAD

**Issue by JanX2** [nvkelso/color-oracle#15]
If you want to apply the color filtering to more than just the desktop background in macOS Catalina (10.15) or later, you need to enable the “Screen Recording Permission”. This wasn’t obvious when I first ran it.

For a detailed description, see:
[How do I configure Screen Recording permission on macOS Catalina?](https://apple.stackexchange.com/questions/384228/how-do-i-configure-screen-recording-permission-on-macos-catalina)

**Comment by nvkelso** [comment]
So this would be a documentation fix?

**Comment by JanX2** [comment]
Yes. In the meantime. I’m evaluating an actual fix. It may take a while to find the time.

It involves enabling all the security stuff Apple has added in recent years (done; can share if requested), detecting the permission setting and either telling the user it’s not set or behaving normally with it set.

**Comment by JanX2** [comment]
#16 should fix this issue.

**Comment by mroge02** [comment]
Upgrading to Sonoma (14.0) seem to require re-enabling Screen Recording.

Before upgrading ColorApp was working (I assume Screen Recording was enabled, but can't check now).

After upgrading, ColorApp ran but did not change the screen. Screen Recoding for ColorApp was enabled. (!)

I disabled Screen Recording and then re-enabled it.  I restarted ColorApp. Now it works fine.
